### DISCUSSION TITLE
index "C:\users\AppData\roaming\ord\index.redb" apears to have been built with older, incompatible version of ord, consider deleting and rebuilding the index: index schema 24, ord schema 25

### DISCUSSION THREAD

**Discussion by Horianu** [ordinals/ord#3624] [Q&A]
index "C:\users\AppData\roaming\ord\index.redb" apears to have been built with older, incompatible version of ord, consider deleting and rebuilding the index: index schema 24, ord schema 25

This is what I get when updating from 0.17.1 to any version of 0.18 , have asked and other didn't have that problem at all.

Any idea why?

**Comment by raphjaph** [comment]
https://docs.ordinals.com/guides/reindexing.html
